__tests__/app.test.js

```diff
@@ -0,0 +1,65 @@
+const app = require('../app')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const db = require('../db/connection')
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    db.end();
+})
+
+describe('GET: 200 /api/categories', () => {
+    test('responds with an array of category objects with the properties: slug and description', () => {
+        const testResult = [
+            {
+              slug: 'euro game',
+              description: 'Abstact games that involve little luck'
+            },
+            {
+              slug: 'social deduction',
+              description: "Players attempt to uncover each other's hidden role"
+            },
+            {
+              slug: 'dexterity',
+              description: 'Games involving physical skill'
+            },
+            {
+              slug: "children's games",
+              description: 'Games suitable for children'
+            }
+          ];
```

Comment:
Make your test more dynamic, dont hard code values. Think of other ways you can test your result.